telco-core/other/sctp_module_mc.yaml

```diff
@@ -3,23 +3,22 @@
 apiVersion: machineconfiguration.openshift.io/v1
 kind: MachineConfig
 metadata:
+  name: load-sctp-module
   labels:
     machineconfiguration.openshift.io/role: worker
-  name: load-sctp-module
 spec:
   config:
     ignition:
-      version: 2.2.0
+      version: 3.2.0
     storage:
       files:
-        - contents:
+        - path: /etc/modprobe.d/sctp-blacklist.conf
+          mode: 0644
```

Comment:
doesnt matter much, but there's no reason having these files read by group or other

Reply:
It appears that the controller/webhook converts 0644 to dec 420. Our reference should reflect the value as it will be on the cluster. @elhadjici can you update to use the decimal value please